website_menu_icon/static/src/js/content.js

```diff
@@ -0,0 +1,195 @@
+odoo.define("website_menu_icon.menu_entry_dialog", function(require) {
+    "use strict";
+
+    var contentMenu = require("website.contentMenu");
+    var MediaDialog = require("wysiwyg.widgets.MediaDialog");
+    var Dialog = require("web.Dialog");
+    var core = require("web.core");
+
+    var MyMenuEntryDialog = contentMenu.MenuEntryDialog.extend({
```

Comment:
All will be much easier if you just `include` instead of `extend`ing this class:

```suggestion
    contentMenu.MenuEntryDialog.include({
```

This way, all `MenuEntryDialog` will automatically have your new code. This way, you can override the `save` event or any other you need to obtain the desired missing values.

Don't forget to call super in all overriden methods. Breaking inheritance is usually :no_entry_sign:.

Reply:
I remember initially doing that way. Than I experimented for many hours with template inheritance and tried everything I could. So this is probably some remaining artifact of that. Will change that.